src/app/package-json.js

```diff
@@ -0,0 +1,303 @@
+import _ from 'lodash';
+import util from 'util';
+
+const KARMA_BROWSERS = ['chrome', 'firefox', 'ie', 'safari'];
+
+/**
+ * Takes advantage of the way V8 orders object properties - by their
+ * assignment order - to "sort" an object in alphabetic order.
+ *
+ * @param  {Object} source
+ * @return {Object}
+ *         A new ordered object.
+ */
+const alphabetizeObject = (source) =>
+  _.pick(source, Object.keys(source).sort());
+
+/**
+ * Identical to `alphabetizeObject`, but there is special handling to
+ * preserve the ordering of "pre" and "post" scripts next to their
+ * core scripts. For example:
+ *
+ *    "preversion": "...",
+ *    "version": "...",
+ *    "postversion": "..."
+ *
+ * @param  {Object} source
+ * @return {Object}
+ *         A new ordered object.
+ */
+const alphabetizeScripts = (source) => {
+  let keys = Object.keys(source);
+  let prePost = keys.filter(
+    k => _.startsWith(k, 'pre') || _.startsWith(k, 'post')
```

Comment:
But lodash is used all over the place, so not much point. Maybe this can be a future refactor.